code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -1,866 +1,810 @@
+			refresh=1
+			if(new_id)
+				suffix = new_id
+				name = "\improper Mulebot ([suffix])"
+				update_controls()
+
+		if("sethome")
+			refresh=0
+			var/new_home = stripped_input(user, "Enter new home tag", "Mulebot [suffix ? "([suffix])" : ""]", home_destination)
+			refresh=1
+			if(new_home)
+				home_destination = new_home
+				update_controls()
+
+		if("unload")
+			if(load && mode != BOT_HUNT)
+				if(loc == target)
+					unload(loaddir)
+				else
+					unload(0)
+
+		if("autoret")
+			auto_return = !auto_return
+
+		if("autopick")
+			auto_pickup = !auto_pickup
+
+		if("report")
+			report_delivery = !report_delivery
+
+		if("close")
+			usr.unset_machine()
+			usr << browse(null,"window=mulebot")
+
+
+
+// returns true if the bot has power
+/mob/living/simple_animal/bot/mulebot/proc/has_power()
+	return !open && cell && cell.charge > 0 && wires.HasPower()
+
+/mob/living/simple_animal/bot/mulebot/proc/buzz(type)
+	switch(type)
+		if(SIGH)
+			audible_message("[src] makes a sighing buzz.", "<span class='italics'>You hear an electronic buzzing sound.</span>")
+			playsound(loc, 'sound/machines/buzz-sigh.ogg', 50, 0)
+		if(ANNOYED)
+			audible_message("[src] makes an annoyed buzzing sound.", "<span class='italics'>You hear an electronic buzzing sound.</span>")
+			playsound(loc, 'sound/machines/buzz-two.ogg', 50, 0)
+		if(DELIGHT)
+			audible_message("[src] makes a delighted ping!", "<span class='italics'>You hear a ping.</span>")
+			playsound(loc, 'sound/machines/ping.ogg', 50, 0)
+
+
+// mousedrop a crate to load the bot
+// can load anything if emagged
+/mob/living/simple_animal/bot/mulebot/MouseDrop_T(atom/movable/AM, mob/user)
+
+	if(user.incapacitated() || user.lying)
+		return
+
+	if (!istype(AM))
+		return
+
+	load(AM)
+
+// called to load a crate
+/mob/living/simple_animal/bot/mulebot/proc/load(atom/movable/AM)
+	if(load ||  AM.anchored)
+		return
+
+	if(!isturf(AM.loc)) //To prevent the loading from stuff from someone's inventory or screen icons.
+		return
+
+	var/obj/structure/closet/crate/CRATE
+	if(istype(AM,/obj/structure/closet/crate))
+		CRATE = AM
+	else
+		if(wires.LoadCheck())
+			buzz(SIGH)
+			return		// if not emagged, only allow crates to be loaded
+
+	if(CRATE) // if it's a crate, close before loading
+		CRATE.close()
+
+	if(isobj(AM))
+		var/obj/O = AM
+		if(O.buckled_mob || (locate(/mob) in AM)) //can't load non crates objects with mobs buckled to it or inside it.
+			buzz(SIGH)
+			return
+
+	if(isliving(AM))
+		if(!load_mob(AM))
+			return
+	else
+		AM.loc = src
+
+	load = AM
+	mode = BOT_IDLE
+	update_icon()
+
+/mob/living/simple_animal/bot/mulebot/proc/load_mob(mob/living/M)
+	if(M.buckled)
+		return 0
+	/*
+	var/turf/T = get_turf(src)
+	if(M.loc != T)
+		density = 0
+		var/can_step = step_towards(M, T)
+		density = 1
+		if(!can_step)
+			return 0
+	*/
+	if(M.client)
+		M.client.perspective = EYE_PERSPECTIVE
+		M.client.eye = src
+	passenger = M
+	load = M
+	M.loc = src
+	return 1
```

Comment:
So you can't get on top of the mulebot anymore, you're inside of it? It's not consistent with how crates are loaded.